src/thor/main.py

```diff
@@ -60,32 +62,32 @@ async def get_release_tasks(release_id: int, task_id: int):
     Currently, it fails without support if either release_id or task_id are not ints, 
     so this should be addressed when appropriate. """
     tasks_to_return = []
-    tKeyList = getTKeys()
+    task_key_list = get_task_keys()
 
-    for id in tKeyList:
-        if readTask(id).get_release_id() == release_id:
-            if id == task_id:
-                tasks_to_return.append(readTask(id))
+    for key in task_key_list:
+        if read_task(key).get_release_id() == release_id:
+            if key == task_id:
+                tasks_to_return.append(read_task(key))
     log.info(f"Successfully obtained task info for {release_id}, {task_id}. ")
     return tasks_to_return
 
 
 @app.get("/tasks")
-async def getTasks():
+async def tasks():

```

Comment:
Especially for APIs, prefer to have functions in a verb-noun pattern. For this, probably `get_tasks` or `get_all_tasks` is better.